OracleOpenJDK/14/Dockerfile

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle OpenJDK 14
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+#
+# (1) openjdk-14.xx_linux-x64_bin.tar.gz
+#     Download from https://jdk.java.net/14/
```

Comment:
There is no click-through required, so your Dockerfile should just use `curl` to download and validate this file automatically.